### ISSUE TITLE
Discrepancy between documentation and behavior of dialogue box

### ISSUE THREAD

**Issue by Kazuo256** [uspgamedev/backdoor#1038]
```lua
-- newpage - Create a new page after this tag, waiting an amount of time
--  wait - how long to wait on current page before going to next
--    <any number value> - will wait this much time
```

But the parameter name expected is "value":

```lua
elseif header == "newpage" then
    if aux_att["value"] and tonumber(aux_att["value"]) then
      local wait_amount = tonumber(aux_att["value"])
```

**Comment by rilifon** [comment]
ops hehe

Em qui, 6 de jun de 2019 às 11:21, Wilson Kazuo Mizutani <
notifications@github.com> escreveu:

**Comment by rilifon** [comment]
Fixed via 7a15190b7e71f4c1b140e52183d3283a3ddb5292
